cpu/stm32/periph/Makefile

```
@@ -49,6 +49,15 @@ ifneq (,$(filter periph_rtt,$(USEMODULE)))
   endif
 endif
 
+# Select the correct implementation for `periph_usbdev`
+ifneq (,$(filter periph_usbdev,$(USEMODULE)))
+  ifneq (,$(filter $(CPU_FAM),f0 f1 f3 l0 l1 l4 l5 g4 wb))
+    SRC += usbdev_fs.c
+  else
+    SRC += usbdev_otg.c
+  endif
```

Maybe use the opposite logic, since otg correspond to less CPUs families (and also older):

```suggestion
  ifneq (,$(filter $(CPU_FAM),f2 f4 f7))
    SRC += usbdev_otg.c
  else
    SRC += usbdev_fs.c  # f0 f1 f3 l0 l1 l4 l5 g4 wb
  endif
```

There's no usbdev fs on g0 ?

---

There is one but again, not the very same IP so I don't even know if this driver can be used there...